### ISSUE TITLE
[BUG] Platform not reachable

### ISSUE THREAD

**Issue by nikitabaldan** [netdata/netdata-cloud#359]
**Describe the bug**	
Cannot login into the platform, to monitor the nodes.

**To Reproduce**	
https://app.netdata.cloud/?utm_source=website&utm_content=top_navigation_sign_up not reachable

**Expected behavior**	
Actually login and monitor our nodes

**Desktop (please complete the following information):**	
 - OS: [e.g. iOS] MacOS Big Sur
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22]	Up to date

**Comment by dionysis-netdata** [comment]
Hi @nikitabaldan ,

still facing the issue? is [https://app.netdata.cloud](https://app.netdata.cloud) not loading at all? All our services are working fine and no other user has reported such issues.

Maybe you can try from a different browser

Let me know if works

**Comment by nikitabaldan** [comment]
Hi @dionysis-netdata, 
It was an italian networking problem (probably with DNS). 
I have faced another problem regarding the configuration of Netdata with docker compose. The container have continued to stop and the logs showed the following message error: `error adding seccomp filter rule for syscall clone3: permission denied`.  To solve that problem I have added `seccomp:unconfined` to the `security_opt` of the Netdata container.

Hope this will help.
